tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-loader.html

```diff
@@ -222,12 +222,31 @@
         this.requestManager.request(url).then(updateSteps);
       },
       _createStepDatum(audioMetadata) {
-        let url = tf_backend.getRouter().pluginRoute('audio', '/individualAudio');
+        const isDemoMode = tf_backend.getRouter().isDemoMode();
+        const searchParam = new URLSearchParams(audioMetadata.query);
         // Include wall_time just to disambiguate the URL and force
         // the browser to reload the audio when the URL changes. The
         // backend doesn't care about the value.
-        url = tf_backend.addParams(url, {ts: audioMetadata.wall_time});
-        url += '&' + audioMetadata.query;
+        searchParam.append('ts', audioMetadata.wall_time);
+        const queryParam = Array.from(searchParam.entries())
+            .reduce((reduced, [key, value]) => {
+              if (reduced[key]) {
+                if (!Array.isArray(reduced[key])) {
+                  reduced[key] = [reduced[key]];
+                }
+                reduced[key].push(value);
+              } else {
+                reduced[key] = value;
+              }
+              return reduced;
+            }, {});
+        const url = tf_backend.getRouter().pluginRoute(
+            'audio',
+            '/individualAudio',
+            queryParam,
+            // customExt is used in demo mode where all routes are all
+            // automatically appended with .json. We need '.wav' for audio.
+            isDemoMode ? '.wav' : '');
```

Comment:
The extension is already ignored unless we're in demo mode, so simpler to just pass '.wav' all the time and let Router figure it out.